README.md

```diff
@@ -140,6 +140,12 @@ network protocols, such as the
 [PixelPusher implementation](https://github.com/hzeller/rpi-matrix-pixelpusher)
 (common in light art installations).
 
+If you have arduino code that uses RGBPanels, you'll likely hit limits around 128x64
```

Comment:
Keep it concise like the other entries.

e.g.

[FastLED/GFX Bridge ](http://marc.merlins.org/perso/arduino/post_2020-01-01_Running-FastLED_-Adafruit_GFX_-and-LEDMatrix-code-on-High-Resolution-RGBPanels-with-a-Raspberry-Pi.html) for Arduino code (porting Arduino code for these to Raspberry Pi).

Reply:
Ok, you got me, terse is not my middle name :)
How about this new version?